### DISCUSSION TITLE
What is  the proper way to update ports in this case?

### DISCUSSION THREAD

**Discussion by eclipse0922** [microsoft/vcpkg#34153] [Q&A] [answered]
Currently I am trying to update Eigen's port.

 Eigen's upstream commit removed some config variables that rbdl port depends on 
(https://gitlab.com/libeigen/eigen/-/commit/f2984cd0778dd0a1d7e74216d826eaff2bc6bfab)

EIGEN_INCLUDE_DIR' is removed, so rbdl is not built and an error occurs.

But I need to update to a version later than that commit, so I can't use the earlier commit.

In this case, rbld should be fixed in the same PR for Eigen?

**Comment by Neumann-A** [comment, marked as answer]
> EIGEN_INCLUDE_DIR' is removed, so rbdl is not built and an error occurs.

patch it back? (is probably the easiest solution)

> In this case, rbld should be fixed in the same PR for Eigen?

CI needs to be green. How you do it is up to you. You can also path rbld to use the Eigen3 targets but be aware that consumers also need to find that target.

**Reply by eclipse0922** [reply]
Thank you for the answer,
